docs/src/intervention.html.jinja2

```diff
@@ -444,8 +444,7 @@
                         <div class="col-md-6">
                             <label>Was there a conversion of VATS to open?</label>
                             <div class="form-group form-inline">
-                                {{ f.inlineRadio("rbvto", "rbvto-n", "n", "No", true) }}
-                                {{ f.inlineRadio("rbvto", "rbvto-y", "y", "Yes", true) }}
+                                {{ f.inlineRadioNY("rbvto") }}
```

Comment:
All these replacements you did need the second parameter to be "true" to indicate that the field is required. I'd recommend you DIFF the final HTML against master branch before committing.